src/main/scala/Game.scala

```diff
@@ -0,0 +1,94 @@
+class Game {
+  var player1: Player = _
+  var player2: Player = _
+
+  def startGame(): Unit = {
+    menu()
+
+    val roundsStr = scala.io.StdIn.readLine("How many rounds? ")
+    val rounds = roundsStr.toInt
```

Comment:
why not have this just as 

val rounds = scala.io.StdIn.readLine("How many rounds? ").toInt